packages/sdk/src/mcp/triggers/api.ts

```diff
@@ -211,6 +211,80 @@ export const createTrigger = createTool({
   },
 });
 
+export const updateTrigger = createTool({
```

Comment:
When I have create/update methods I really expect to use the same underlying implementation and changing only insert to update or even better use upsert but validation/etc should be exactly the same. So I'd like to see something like 

handler: upsertTrigger

which is the same function as create, if possible